api/src/services/items.ts

```diff
@@ -260,23 +264,36 @@ export class ItemsService<Item extends AnyItem = AnyItem> implements AbstractSer
 	 * Create multiple new items at once. Inserts all provided records sequentially wrapped in a transaction.
 	 */
 	async createMany(data: Partial<Item>[], opts?: MutationOptions): Promise<PrimaryKey[]> {
-		const primaryKeys = await this.knex.transaction(async (trx) => {
+		const { primaryKeys, emitActions } = await this.knex.transaction(async (trx) => {
```

Comment:
Can we rename `emitActions` to `nestedActionEvents` for consistency with other similar usages? Another similar renaming in line 289 from actionEvent into nestedActionEvent.

Reply:
✔️